examples/MySQL-client/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ project(MySQL-client-example)
 hunter_add_package(MySQL-client)
 find_package(MySQL-client REQUIRED)
 
+if(MSVC) #LIBCMT conflicts with MSVCR
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /NODEFAULTLIB:LIBCMT.lib")
```

Comment:
Looks like you are right, I didnt see it originally but MySQL has the following:

     IF(NOT WINDOWS_RUNTIME_MD)
         FOREACH(flag 
          CMAKE_C_FLAGS_RELEASE    CMAKE_C_FLAGS_RELWITHDEBINFO 
          CMAKE_C_FLAGS_DEBUG      CMAKE_C_FLAGS_DEBUG_INIT 
          CMAKE_CXX_FLAGS_RELEASE  CMAKE_CXX_FLAGS_RELWITHDEBINFO
          CMAKE_CXX_FLAGS_DEBUG    CMAKE_CXX_FLAGS_DEBUG_INIT)
          STRING(REPLACE "/MD"  "/MT" "${flag}" "${${flag}}")
          STRING(REPLACE "/Zi"  "/Z7" "${flag}" "${${flag}}")
     etc..

I can't see where WINDOWS_RUNTIME_MD would be defined, so I am guessing this is always going to run. If I alter the example to use

    set(CMAKE_CXX_FLAGS_RELEASE "/MT")
    set(CMAKE_CXX_FLAGS_DEBUG "/MTd")

it compiles fine.